### PR TITLE
Feature/27837 Assign forms UI scaffolding

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- Added routes/views for displaying check windows and available check forms for assignment
- Added methods responsible for fetching current or future check windows and check forms based on type (live | familiarisation)
- Added new view to reflect assigned forms by type (live | familiarisation) on the UI level

### CODE REVIEW

File: admin/data/sql/migrations/20190110132657.do.create-view-check-windows-with-status-and-form-count-by-form-type.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
+CREATE VIEW [mtc_admin].[vewCheckWindowsWithStatusAndFormCountByFormType] AS
```
Comment (jon-shipley) [inline admin/data/sql/migrations/20190110132657.do.create-view-check-windows-with-status-and-form-count-by-form-type.sql]: Form Count By Form Type - I suspect we could drop the second 'form' and just call it `vewCheckWindowsWithStatusAndFormCountByType`

File: admin/views/check-form/view-assign-forms-to-check-windows.ejs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <p>Check Window</p>
+    </span>
+    <div class="assigned-check-windows">
+      <% checkWindowData.forEach(cw => { %>
+      <dl>
+        <dd>
+          <p class="font-small"><%= cw.name %></p>
+          <div><span class="bold-small"><a href="/check-form/select-form/familiarisation/<%= cw.urlSlug %>">'Try it out' period</a></span>
+            (<%= cw['FamiliarisationCheckFormCount'] === 1 ? `${cw['FamiliarisationCheckFormCount']} form` : `${cw['FamiliarisationCheckFormCount']} forms` %> assigned)</div>
+          <p><%= cw.familiarisationCheckStartDate.format('D MMMM') %> to <%= cw.familiarisationCheckEndDate.format('D MMMM YYYY') %></p>
```
Comment (jon-shipley) [inline admin/views/check-form/view-assign-forms-to-check-windows.ejs]: I think the formatted date should be passed in from the Presenter.  Also shoudn't the date formatting should be a method in the date service?

File: admin/helpers/check-form-presenter.js [hunk 3]
```diff
@@ -54,4 +54,42 @@ checkFormPresenter.getFlashMessageData = (uploadData) => {
...
+ * @param {Array} checkWindows
+ * @returns {Array}
+ */
+checkFormPresenter.getPresentationCheckWindowListData = (checkWindows) => {
+  const checkWindowData = []
+  checkWindows.forEach(cw => {
+    checkWindowData.push({
+      name: cw.name,
+      urlSlug: cw.urlSlug,
+      familiarisationCheckStartDate: cw.familiarisationCheckStartDate.format('D MMMM'),
```
Comment (jon-shipley) [inline admin/helpers/check-form-presenter.js]: Why aren't these calls using the dateService?

Comment (GeorgeChatzigiannis) [reply admin/helpers/check-form-presenter.js]: @jon-shipley updated based on our discussion

**Pull request merged**